src/app/ui/home/home.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="container">
+  <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+    <ol class="carousel-indicators">
+      <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+      <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+      <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+    </ol>
+    <div class="carousel-inner shadow-lg bg-white rounded">
```

Comment:
Regarding the names of the images, please create a directory inside `assets/images` which is called homepage and add the images there. Also, rename each image to describe it's content, and not 1,2,3...